### DISCUSSION TITLE
How to add line feed after the use statement

### DISCUSSION THREAD

**Discussion by KarChung-Konstant** [PHP-CS-Fixer/PHP-CS-Fixer#8126] [Q&A] [answered]
My code so far:
```
use Orange;
use Apple;
public $fillable = [];
```

Expected result:
```
use Orange;
use Apple;

public $fillable = [];
```

**Comment by dmitryuk** [comment]
@KarChung-Konstant your code can't be compiled, provide correct

**Reply by Wirone** [reply]
Thank you for encouraging the author to provide correct code snippet 👍. Anyway, it was the `class` wrapping that was missing, as provided code contained traits usage and a property.

**Comment by Wirone** [comment, marked as answer]
@KarChung-Konstant please look at [`class_attributes_separation`](https://cs.symfony.com/doc/rules/class_notation/class_attributes_separation.html), especially `property` element.

<img width="905" alt="image" src="https://github.com/user-attachments/assets/fcc2f23f-0bc3-49b6-8b2d-4b6b7aa6eb98">
